ComfortZoneApp/app/src/main/res/menu/menu_bottom_navigation.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
+        android:id="@+id/action_home"
```

Comment:
wouldn't this be the flight tab? does it make more sense to name it as action_flight?